root/base/release.tx

```diff
@@ -84,6 +84,13 @@
     <li>
       <a rel="noopener nofollow" href="http://cpants.cpanauthors.org/release/[% $release.author %]/[% $release.name %]"><i class="fa fa-fw fa-line-chart black"></i>Kwalitee</a>
     </li>
+    %% if defined $distribution.river.bus_factor {
+    <li>
+      <div id="tooltip" data-toggle="tooltip" data-placement="right" title="The # people with an indexing permission on [% $release.distribution %] who have released something to CPAN in the last 2 years">
```

Comment:
This id seems wrong, but it would be better to remove the id and add the class `ttip`. Code in `cpan.js` will then set up the tooltip without needing and `<script>` tags or anything page specific.